wdl_runner/cromwell_launcher/setup.sh

```diff
@@ -20,32 +20,67 @@
 set -o errexit
 set -o nounset
 
+# retry_cmd
+#
+# Any operation that is dependent on internet connectivity and server
+# availability is vulnerable to intermittent failure.
+#
+# This function will retry the specified command up to a specified number
+# of attempts (default 5 attempts).
+# The wait between attempts can also be set (default: 5 seconds).
+function retry_cmd () {
+  local cmd="${1}"
+  local max_attempts="${2:-5}"
+  local wait_seconds="${3:-5}"
```

Comment:
Maybe it would be good to specify how one can provide a custom number other than 5.